msteams-platform/bots/how-to/conversations/send-proactive-messages.md

```diff
@@ -86,11 +86,43 @@ See the following video to learn how to send proactive message from bots:
 
 Sending proactive messages to the users is an effective way to communicate with your users. However, from the user's perspective, the message appears unprompted. If there's a welcome message, it will be the first time that they’ve interacted with your app. It's important to use this functionality and provide the complete information to the user to understand the purpose of this message.
 
+> [!NOTE]
```

Comment:
Don't put this info in a note. Let's use note as less as we can in the docs.
Mention it in a sub-section of its own in this article. (And that sub-section is not best practices.)